### PR TITLE
Loc update

### PR DESCRIPTION
**Pull request by jaredthomas68**
This PR updates necessary files to move the location of 01 and 02 designs to correspond with other related work. This PR should be reviewed and merged after #2

### CODE REVIEW

File: reference-systems/03-gulf-of-mexico-hydrogen/greenHEART/speed_dir_data.csv
Comment (kbrunik) [inline reference-systems/03-gulf-of-mexico-hydrogen/greenHEART/speed_dir_data.csv:1]: let's remove this speed file

File: reference-systems/04-new-york-bight-hydrogen/greenHEART/input-files/plant/default_fin_config.yaml
Comment (kbrunik) [inline reference-systems/04-new-york-bight-hydrogen/greenHEART/input-files/plant/default_fin_config.yaml:1]: I don't really understand what the change is here but seems okay?

File: reference-systems/03-gulf-of-mexico-hydrogen/greenHEART/input-files/plant/default_fin_config.yaml [hunk 1]
```diff
@@ -38,4 +38,4 @@ cp_capacity_credit_percent: [0]
 degradation: [0]
 revenue:
   ppa_price_input: [0.0] # cents/kWh
-  ppa_escalation: 0 # %
\ No newline at end of file
+  ppa_escalation: 0 # %
```
Comment (kbrunik) [inline reference-systems/03-gulf-of-mexico-hydrogen/greenHEART/input-files/plant/default_fin_config.yaml:41]: A little confused about what changed here but doesn't seem to be a problem

**Review by kbrunik — CHANGES_REQUESTED** [review]

File: reference-systems/03-gulf-of-mexico-hydrogen/greenHEART/speed_dir_data.csv
Comment (jaredthomas68) [reply reference-systems/03-gulf-of-mexico-hydrogen/greenHEART/speed_dir_data.csv:1]: I thought I had. Thanks for catching it.

**Review by kbrunik — APPROVED** [review]

File: reference-systems/04-new-york-bight-hydrogen/greenHEART/input-files/plant/default_fin_config.yaml
Comment (jaredthomas68) [reply reference-systems/04-new-york-bight-hydrogen/greenHEART/input-files/plant/default_fin_config.yaml:1]: I don't know either. It should be no change, but is registering for some reason. This is where there was a conflict before, so maybe I changed the type of white space or something

**Pull request merged**